RevisionNotes/MIT_Lec_15.md

```diff
@@ -0,0 +1,29 @@
+# 15 - Projections into subspaces
+
+### The basics of projection
+![Projection image](./Images/MIT_15_projection.png)
+
+### Defining a basic formula for projecting b onto a line formed by a single vector.
+![Projection derivation example](./Images/MIT_15_basic.png)
+
+- Let `p` be the projection of `b` onto the line going through `a`.
+- Since `p` is at the shortest distance possible from `b`, the line joining them must me perpendicular to the line formed by multiples of `a`.
+- Therefore, if `e = b - p`, then e must be perpendicular to a.
+- Now `p` is a scalar multiple of `a`, so let `p = ax` with x being a scalar.
+- We get `a.t (b- ax) = 0`.
+- Rearranging and substituting x, `p = a(a.t . b / a.t . a)`
+- By using the associative law, we see that `p = (a . a.t / a.t . a)b` with the first bracket being known as the projection vector.
```

Comment:
AFAIK, first bracket is projection matrix, right?
(Ref : https://ocw.mit.edu/courses/mathematics/18-06sc-linear-algebra-fall-2011/least-squares-determinants-and-eigenvalues/projections-onto-subspaces/MIT18_06SCF11_Ses2.2sum.pdf)

Reply:
My bad, changed it to projection matrix.

Reply:
Gentle ping @kkothari2001

Reply:
Done, thanks.